apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -118,6 +119,7 @@ class Courses extends Component {
             {/* English, student.  (Also the default to be shown when signed out.) */}
             {isEnglish && !isTeacher && (
               <div className={'announcements'}>
+                <SpecialAnnouncement isTeacher={isTeacher} />
```

Comment:
When do we use SpecialAnnouncement vs MarketingAnnouncementBanner?

Reply:
I think SpecialAnnouncement was related to Covid learning from home so it was for something less evergreen than MarketingAnnouncementBanner? This was all set up before my time though so someone else might know the reasoning behind the use cases for each.

This is what was originally on this page so I went with it as opposed to MarketingAnnouncementBanner.